### DISCUSSION TITLE
Bedleveling..  Sequence.

### DISCUSSION THREAD

**Discussion by apr1967** [Jyers/Marlin#641] [Q&A] [answered]
I think I know the answer, but then again I am not sure.

I think I should Set my Z, then manually bed level.  Perhaps re set Z and then run the auto bed level?

Just want to make sure I follow the best procedure.

**Comment by vhristenco** [comment]
I use this sequence: manual leveling -> adjust Z -> Level -> save

**Reply by duraluminium** [reply]
You can't manually level if your Z offset is a fresh 0 after flashing the new firmware. In that state you have almost 3mm clearance nozzle-bed. You HAVE TO set a Z offset first before you can manually level with paper!

**Comment by Jyers** [comment, marked as answer]
That sounds about right.

**Reply by duraluminium** [reply]
You can't manually level if your Z offset is a fresh 0 after flashing the new firmware. In that state you have almost 3mm clearance nozzle-bed. You HAVE TO set a Z offset first before you can manually level with paper!
